validphys2/src/validphys/dataplots.py

```diff
@@ -727,12 +723,11 @@ def plot_training_validation(fit, replica_data, replica_filters=None):
 
     """
     training, valid = zip(*((dt.training, dt.validation) for dt in replica_data))
-    fig, ax = plt.subplots(
-        figsize=(
-            max(plt.rcParams.get("figure.figsize")),
-            max(plt.rcParams.get("figure.figsize")),
-        )
-    )
+    fig, ax = plotutils.subplots(figsize=(
+            max(mpl.rcParams.get("figure.figsize")),
+            max(mpl.rcParams.get("figure.figsize")),
+        ))
+    
```

Comment:
Could you run black on the changes? This one will probably look a bit more like in the previous version:

```suggestion
    fig, ax = plotutils.subplots(
        figsize=(
            max(mpl.rcParams.get("figure.figsize")),
            max(mpl.rcParams.get("figure.figsize")),
        )
    )
```